plugins/by-name/blink-cmp/default.nix

```
@@ -131,4 +155,11 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
       };
     };
   };
+
+  extraConfig = cfg: {
+    warnings = lib.optional (cfg.settings ? documentation) ''
+      Nixvim(plugins.blink): settings.documentation was not properly defined, use settings.windows.documentation
+      instead
```

```suggestion
      Nixvim(plugins.blink): `settings.documentation` does not correspond to a known setting, use `settings.windows.documentation` instead.
```

1. Slight re-wording suggestion (if you agree)
2. Ideally, put everything on one line

Warnings get printed verbatum to stdout, without any markdown formatting. Therefore (unlike option descriptions) we can't rely on markdown to un-wrap lines that we would prefer to have wrapped, if we don't want them to be printed on multiple lines.